build.sbt

```diff
@@ -132,6 +133,8 @@ val docs = project
     micrositeGithubRepo := "cats-retry",
     micrositeGitterChannel := true,
     micrositeTwitterCreator := "@cbirchall",
+    micrositeCompilingDocsTool := WithMdoc,
+    mdocIn := (sourceDirectory in Compile).value / "tut",
```

Comment:
If you manage to get this working, let's rename `src/tut` to `src/mdoc` to avoid confusion.